pixi.toml

```diff
@@ -282,11 +289,14 @@ py-build-notebook = { cmd = "pip install -e rerun_notebook", depends_on = [
 # Dedicated alias for building the python bindings for the `py` environment.
 py-build = "pixi run -e py py-build-common"
 
+# Dedicated alias for building the python bindings in release modefor the `py` environment.
+py-build-release = "pixi run -e py py-build-common-release"
```

Comment:
Not sure if there's anything we can do about it, but as an observation, this is very very similar to:
```
pixi run py-build --release
```
which almost works, but still depends on `rerun-build` instead of `rerun-build-release`.

Reply:
yeah that's the reason I didn't go that way - there doesn't seem to be a way to pass arguments through to dependent commands